PlaceRouteHierFlow/PnRDB/Makefile

```diff
@@ -6,7 +6,7 @@ SRCS=$(OBJS:.o=.cpp)
 all: depend unit_tests PnRDB.a
 
 depend:
-	$(CXX) $(CXXFLAGS) -E -MM $(SRCS) > .depend
+	@$(CXX) $(CXXFLAGS) -E -MM $(SRCS) > .depend
```

Comment:
What does the @ do?

Reply:
It quiets the command.  Since it is a maintenance command that always runs it's common to quiet it -- e.g. if make runs and doesn't trigger any builds, it's preferable to have it just return quietly.